docs/examples/Rocket_Car.rst

```diff
@@ -0,0 +1,1585 @@
+##
+
+.. raw:: html
+
+   <center>
+
+:math:`\mathbf{\text{Modeling of a Variable-Mass Nonholonomic Gyrostatic Rocket Car Using Extended Kane's Equations}}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Ge, Z.M. and Cheng, Y.H., 1982. Extended Kane’s equations for nonholonomic variable mass system.}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Figure 1: Idealized model of a jet racing car that is propelled by a rocket engine at point P, the rocket engine being treated as a variable mass particle at P}`
```

Comment:
Figures need to use the proper rst syntax:

https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure